src/ts/uimanager.ts

```diff
@@ -762,7 +762,7 @@ class InternalUIInstanceManager extends UIInstanceManager {
 /**
  * Extended interface of the {@link Player} for use in the UI.
  */
-interface WrappedPlayer extends PlayerAPI {
+export interface WrappedPlayer extends PlayerAPI {
```

Comment:
Do we really want to export this? 🤔

Reply:
We are using the interface in the type guard as one of the player type, when typed in `uimanager`